src/kv/bulk.rs

```diff
@@ -33,6 +33,7 @@ fn bulk_api_client(user: &GlobalUser) -> Result<HttpApiClient, failure::Error> {
         config,
         Environment::Production,
     )
+    .map_err(|e| failure::format_err!("{}", e))
```

Comment:
worth considering some prefix to this error here? `[bulk api]` or something that gives a hint in case the trace isn't included (I'm unsure what `failure` will provide without specifically asking for the backtrace)? 

not saying its a must, but since you're already in here...

Reply:
Hmm honestly the better solution is to move to `anyhow`, since that's what `cloudflare-rs` uses. I'll try swapping that out and see what the breakage is like.

Reply:
@nilslice see https://github.com/cloudflare/wrangler/pull/1881